internal/nexodus/nexodus.go

```diff
@@ -271,12 +278,28 @@ func (ax *Nexodus) Start(ctx context.Context, wg *sync.WaitGroup) error {
 		return fmt.Errorf("handleKeys: %w", err)
 	}
 
-	user, _, err := ax.client.UsersApi.GetUser(ctx, "me").Execute()
+	var user *public.ModelsUser
+	err = util.RetryOperation(ctx, 10*time.Second, 3, func() error {
```

Comment:
I think it would be good to put these timer and attempt configuration in the constant.

Reply:
Sure sure, done!